### ISSUE TITLE
Fine-tuning for Custom Dataset

### ISSUE THREAD

**Issue by NidhiSalian** [supernlogn/squeezeDetTL#1]
Really appreciate all the work that's gone into creating this repo. It's super user-friendly and intuitive.

I have a question though - what if I want to fine-tune a pre-trained SqueezeDet model on my own custom dataset with say, 50 classes? Are you planning on adding code to allow for this?

**Comment by supernlogn** [comment]
Hello!
Thank you very much for your kind comments!
What do you mean by "custom" dataset? 
A dataset with your own images and labels using for example the PASCAL VOC annotation 
or
a completely custom dataset using your own type of annotation (not one of the PASCAL, MS COCO, KITTI or even openImages)?

**Comment by NidhiSalian** [comment]
A custom dataset with images of multiple scales. Essentially uses MS COCO-style annotations saved into Tensorflow record format.

**Comment by supernlogn** [comment]
Multiple scales is not an issue. I will do my best to add code/guide for this till Monday.

**Comment by supernlogn** [comment]
Hello,
I just added [this](https://github.com/supernlogn/squeezeDetTL/blob/master/scripts/user_usage.md) guide. I hope it helps! 
If not please tell me and I will see what else I can do.

Have a nice Weekend!

**Comment by NidhiSalian** [comment]
Haha this seems like a pretty great guide! Thanks a lot, I'll let you know if I run into any issues.
